README.md

```diff
@@ -164,6 +168,21 @@ For more information on different components that compose our commit messages, p
 
 Additionally, CI will use `commitlint` to validate the commits associated with a PR in the `Lint and Validate` job.
 
+### Adding Documentation
```

Comment:
nit: can you update this to something like `Generating type interface documentation`